libraryversions.toml

```diff
@@ -292,6 +292,7 @@ WEAR_TILES = { group = "androidx.wear.tiles", atomicGroupVersion = "versions.WEA
 WEAR_WATCHFACE = { group = "androidx.wear.watchface", atomicGroupVersion = "versions.WEAR_WATCHFACE" }
 WEBKIT = { group = "androidx.webkit", atomicGroupVersion = "versions.WEBKIT" }
 WINDOW = { group = "androidx.window", atomicGroupVersion = "versions.WINDOW" }
+WINDOW_CORE = { group = "org.jetbrains.window.core", atomicGroupVersion = "versions.WINDOW", overrideInclude = [ ":window:window-core" ]  }
```

Comment:
"versions.WINDOW" is already `1.6.0-alpha01` in AOSP. Shouldn't we update it here?

Reply:
I didn't touch the version. As I understand the version was set in the commit we based our fork on

Reply:
This is because we don't merge `window` and `libversions` to jb-main, they can be old. The actual state is in `integration`, but as with any other lib we merge only released commits to `jb-main`